student.rb

```diff
@@ -8,4 +8,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     "¯\(ツ)/¯"
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
```

Comment:
- [ ] According to the has-many / belongs-to relationships between Classroom and Student, Getters for the instance variable `@classrom` should be present.